_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,586 @@
+---
+title: ONT FS.com Generic Compatible XGSPON Stick ONU with MAC SFP+ (XGS-ONU-25-20NI)
+has_children: true
+layout: default
+parent: ONT FS.com
+---
+
+# Hardware Specifications
+
+|                  |                                                                            |
+| ---------------- | -------------------------------------------------------------------------- |
+| Vendor/Brand     | FS                                                                         |
+| Model            | XGS-ONU-25-20NI                                                            |
+| ODM              | CIG                                                                        |
+| ODM Product Code | XG-99S                                                                     |
+| Chipset          | Cortina CA8271A                                                            |
+| Flash            | MX35LF1GE4AB 128MB                                                         |
+| RAM              | 128MB                                                                      |
+| CPU              | Taroko V0.2 (MIPS)                                                         |
+| CPU Clock        | 500MHz                                                                     |
+| Bootloader       | SATURN uboot                                                               |
+| System           | Custom Linux by Cortina (Saturn SDK) based on Kernel 4.4 Saturn-sfpplus-r1 |
+| 2.5GBaseX        | Yes                                                                        |
+| XGMII/XSGMII     | Yes                                                                        |
+| Optics           | SC/APC                                                                     |
+| IP address       | 192.168.100.1                                                              |
+| Web Gui          |                                                                            |
+| SSH              | No                                                                         |
+| Telnet           | ✅ user `PON S/N`, password `8digit HMAC-MD5 (PON S/N uppercase)`         |
```

Comment:
as already mentioned this username and password pair is not from telnet but is from enable, so it does not go here